v2/swift/main.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  main.swift
+//  YelpAPISample
+//
+
+import Foundation
+
+// Fill in your API keys here from
+// https://www.yelp.com/developers/manage_api_keys
+let client = YLPClient(consumerKey: "",
+                       consumerSecret: "",
+                       token: "",
+                       tokenSecret: "")
+
+// Create a dispatch group to wait for search results
+let group = dispatch_group_create()
+dispatch_group_enter(group)
```

Comment:
@SSheldon Works for me. Like I said, I don't feel too strongly about it. I'm fine with it going out like this.